FV3/testsuite/data/c48_basic/test_baseline/input.nml

```diff
@@ -11,6 +11,7 @@
  blocksize = 24
  chksum_debug = .false.
  dycore_only = .F.
+ disable_phys_restart_write = .F.
  fdiag = 1.
```

Comment:
These shouldn't be necessary to add, since the default is False.
```suggestion
 fdiag = 1.
```

Reply:
I just added it there for completeness -so that an user could see it as an available option.  I left the default value of .false. so the output behavior is unchanged if one is using the namelist.